projects/1.20-fabric/assets/world-preview/world_preview/lang/zh_cn.json

```
@@ -0,0 +1,88 @@
+  "world_preview.preview.btn-caves": "切换高亮洞穴生物群系",
+  "world_preview.preview.btn-home": "重置预览中心至原点\n\n§7§o不一定是出生点！§r",
+  "world_preview.preview.btn-toggle-structures": "切换所有结构可见性",
+  "world_preview.preview.btn-toggle-structures.disabled": "切换所有结构可见性\n\n§7§o结构采样已禁用，可在设置中启用§r",
+  "world_preview.preview.btn-reset-structures": "重置结构可见性为默认值",
+  "world_preview.preview.btn-reset-structures.disabled": "重置结构可见性为默认值\n\n§7§o结构采样已禁用，可在设置中启用§r",
+  "world_preview.preview.btn-toggle-heightmap": "切换高度图和生物群系地图",
+  "world_preview.preview.btn-toggle-heightmap.disabled": "切换高度图和生物群系地图\n\n§7§o高度采样已禁用，可在设置中启用§r",
+  "world_preview.preview.btn-cycle.BIOMES": "生物群系",
+  "world_preview.preview.btn-cycle.STRUCTURES": "结构",
+  "world_preview.preview.btn-cycle.SEEDS": "种子",
+  "world_preview.preview.btn-cycle.structures.disabled.tooltip": "结构采样已禁用，可在设置中启用",
+  "world_preview.preview-settings.title": "预览设置",
+  "world_preview.preview-display.tooltip": "生物群系：%s\n坐标：%s\n高度：%s",
+  "world_preview.preview-display.tooltip.controls": "生物群系：%s\n方块：%s\n坐标：%s\n\n§7§o控制：\n - X/Z：拖动地图\n - Y：鼠标滚轮§r",
+  "world_preview.preview-display.struct.tooltip":  "结构：%s\n中心：%s",
+  "world_preview.preview-display.struct.tooltip.controls":  "结构：%s\n中心：%s\n\n§7§o控制：\n - X/Z：拖动地图\n - Y：鼠标滚轮§r",
+  "world_preview.settings.title": "世界预览 设置",
+  "world_preview.settings.general.title": "常规",
+  "world_preview.settings.general.head": "世界预览 常规设置",
+  "world_preview.settings.general.threads": "生物群系采样线程数：%d",
+  "world_preview.settings.general.threads.tooltip": "较高的数值能够加快预览\n\n但是，使用§l全部§r可用线程可能会导致线程冻结",
+  "world_preview.settings.general.full.chunk": "一次性采样所有Y层",
+  "world_preview.settings.general.full.chunk.tooltip": "禁用时，仅加载当前可见范围内Y层，而不是完全采样所有Y层\n\n仅单个层采样最初速度更快, 但会导致浏览Y层之间不够流畅",
+  "world_preview.settings.general.background": "后台采样相邻Y层",
+  "world_preview.settings.general.background.tooltip": "启用时，当前激活层完全采样后，将在后台采样相邻Y层\n\n当§l全部§r层被采样后，后台采样将停止\n\n当一次性采样整个Y层后，此项无效果",
+  "world_preview.settings.general.struct": "采样结构",
+  "world_preview.settings.general.struct.tooltip": "是否检测一个区块内§l可能§r出现的结构\n\n请注意，由于结构生成的复杂性，这些结果并非100%准确\n\n§6中等性能影响§r",
+  "world_preview.settings.general.heightmap": "采样高度",
+  "world_preview.settings.general.heightmap.tooltip": "是否为维度生成高度图\n\n§c高性能影响§r",
+  "world_preview.settings.general.controls": "在工具提示中显示控制方式",
+  "world_preview.settings.general.controls.tooltip": "在预览工具提示中显示控制方式",
+  "world_preview.settings.general.frametime": "显示帧时间",
+  "world_preview.settings.general.frametime.tooltip": "显示预览渲染的帧时间",
+  "world_preview.settings.sample.title": "分辨率",
+  "world_preview.settings.sample.head": "高级生物群系采样设置",
+  "world_preview.settings.sample.numChunk.title1": "预览时每个区块的大小",
+  "world_preview.settings.sample.numChunk.title2": "此项控制地图预览的缩放等级",
+  "world_preview.settings.sample.numChunk.name.NUM_16": "16x16 像素（最大）",
+  "world_preview.settings.sample.numChunk.name.NUM_8": "8x8 像素",
+  "world_preview.settings.sample.numChunk.name.NUM_4": "4x4 像素 （推荐）",
```

```suggestion
  "world_preview.settings.sample.numChunk.name.NUM_4": "4x4 像素（推荐）",
```
```suggestion
  "world_preview.settings.sample.numChunk.name.NUM_4": "4x4像素（推荐）",
```
这个4x4和像素之间的空格如果选择删除则推荐所有非格式半角全角交界处的空格也要删除

---

已修正